packages/shared/src/features/onboarding/shared/FunnelStepper.tsx

```diff
@@ -224,29 +229,24 @@ export const FunnelStepper = ({
             disabledEvents={[CheckoutEventNames.CHECKOUT_LOADED]}
             successCallback={successCallback}
           >
-            {funnel.chapters.map((chapter: FunnelChapter) => (
-              <Fragment key={chapter?.id}>
-                {chapter?.steps?.map((funnelStep: FunnelStep) => {
-                  const isActive = funnelStep?.id === step?.id;
-                  const Wrapper = isActive ? Fragment : HiddenStep;
```

Comment:
Changing the wrapping component can also cause a re-mount. Meaning, from inactive to active would re-mount the component apart from mounting on load already.